Added docs for execute action - result status (#27)
CiscoTestAutomation/genie: docs/blitz/design/actions/actions_list.rst

        device: PE1
        timeout: 10

> The user can now change the result status to failed, passed, aborted, blocked, skipped, errored
> and passx, when the step is passed. In the below example we assigned result_status to passx.
> when only this step is passed, the result will be changed to ``passx``.

Reviewer: ```suggestion
The user can now change the result status to failed, passed, aborted, blocked, skipped, errored
or passx when the step is passed. In the example below, result_status is changed to passx.
```

Author: updated!!!

.. code-block:: YAML

    - execute: # ACTION
        # (Either device hostname or device alias)
        device: R1
        # Send show version to the device
        command: show version
        # To Change the result status
        result_status: passx

> In the below example when either include or exclude keyword is passed its result will be changed
> to ``aborted``. 

Reviewer: ```suggestion
In the example below, if include or exclude conditions are met, their results will be changed to ``aborted``. 
```

Author: updated!!!

Reviewer 2: please change from `aborted` to `failed`. usual case would be changing to `passx` or `failed`. `aborted` or `errored`, user can specify, but not usual...

Author: updated
